src/main/java/uk/ac/ebi/embl/flatfile/writer/FlatFileWriter.java

```diff
@@ -140,16 +142,16 @@ public static void writeBlock(Writer writer, String firstLineHeader,
 
 			// Break line at optimal line length if no luck finding a split 
 			// character and we have set to break.
-			if (end == 0) {
-				if (wrapChar == WrapChar.WRAP_CHAR_BREAK)
+			if (end == 0 && forceBreak) {
+				if (customMaxLineLength == null)
```

Comment:
My opinion is to 
1) do not change optimalLineLength but to make max line length as 200

The above condition(if (customMaxLineLength == null)) retains exiting business for all writers which was using WRAP_CHAR_BREAK. because those writers just set forceBreak=true but customMaxLineLength will be null for them.

For CC line also we try to split the line within optimalLineLength if we could not do that then we try to find a way to split the line between optimalLineLength and MaximumLineLength (e.g., between 80 to 200 character). 

if we could not find any way to split line between 80 to 200 character then we force split at 200 character.

For me it looks much safer, as it follows exiting order.